libs/game/sprite.ts

```diff
@@ -732,10 +732,9 @@ class Sprite extends sprites.BaseSprite {
     }
 
     registerObstacle(direction: CollisionDirection, other: sprites.Obstacle) {
-        if (other == undefined) return;
```

Comment:
why this?

Reply:
Ahh I should have mentioned this one, [``TileMap.getObstacle``](https://github.com/microsoft/pxt-common-packages/blob/master/libs/game/tilemap.ts#L249) can no longer return undefined after the physics interpolation changes, so this is now a redundant check. I just noticed that it while moving stuff around above.

Reply:
Cool! Just checking